handbook/company/leadership.md

```diff
@@ -6,7 +6,7 @@ The idea here is to get this vision implemented on a single departmental handboo
 Another thing is that we need to get a better intuitive understanding of who these pages are designed to serve.  So in order to put ourselves in their shoes (get behind their eyeballs), we need something to look at.  So doing a particular page first provides us with that canvas.
 
 - `# Name of department`
-  - "This handbook page details processes specific to working `[with](fleetdm.com/handbook/company/nameOfDepartment#contact-us)` and `[within](fleetdm.com/handbook/company/nameOfDepartment#)` the (Name of dpartment) department." 
+  - "This handbook page details processes specific to working `[with](fleetdm.com/handbook/company/nameOfDepartment#contact-us)` and `[within](fleetdm.com/handbook/company/nameOfDepartment)` the (Name of dpartment) department." 
```

Comment:
1. "Name of dpartment" (missing "e"
2. Wasn't part of this change, but while you're in there, better to use kebab-case for the example dept name rather than camelCase